.travis.yml

```diff
@@ -35,9 +35,15 @@ matrix:
   include:
     - os: linux
       rust: stable
-      env:
-        NAME: clippy
+      name: clippy
       install:
         - rustup component add clippy
       script:
         - cargo clippy --verbose --all --tests
+    - os: linux
+      rust: stable
+      name: deb
+      install:
+        - cargo install cargo-deb
```

Comment:
How long does it install? Maybe cache binaries like we do with cargo-travis? https://github.com/tox-rs/tox/blob/master/.travis.yml#L61-L66